packages/walletconnect/__tests__/utils/in-memory-wallet.ts

```diff
@@ -0,0 +1,176 @@
+import { newKit } from '@celo/contractkit';
+import { toChecksumAddress } from '@celo/utils/lib/address';
+import { Alfajores } from '@celo-tools/use-contractkit';
+import WalletConnect from '@walletconnect/client';
+import { IInternalEvent } from '@walletconnect/types';
+import debugConfig from 'debug';
+
+import {
+  AccountsProposal,
+  CLIENT_EVENTS,
+  ComputeSharedSecretProposal,
+  DecryptProposal,
+  PersonalSignProposal,
+  Request,
+  SignTransactionProposal,
+  SignTypedSignProposal,
+  SupportedMethods,
+} from '../../src/types';
+import {
+  parseComputeSharedSecret,
+  parseDecrypt,
+  parsePersonalSign,
+  parseSignTransaction,
+  parseSignTypedData,
+} from './common';
+
+const debug = debugConfig('in-memory-wallet');
+
+const privateKey =
+  '04f9d516be49bb44346ca040bdd2736d486bca868693c74d51d274ad92f61976';
```

Comment:
Is this okay to be here?

Reply:
it's a privkey generated for testing, it'd be unfortunate if anybody is using that key I suppose...